game/templates/game/lesson_detail.html

```
@@ -351,19 +298,14 @@ <h3>📝 Practice Exercise</h3>
-        action="{% url 'complete_lesson' lesson.title %}"
+    <a
+        href="{% url 'complete_lesson' lesson.title %}"
+        style="text-decoration:none;"
     >
-        {% csrf_token %}
-
-        <button
-            type="submit"
-            class="complete-btn"
-        >
+        <button class="complete-btn">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)